commands/builds/create.js

```diff
@@ -25,11 +25,12 @@ function compressSource (context, dir, tempFile) {
 
     if (tarVersion.match(/GNU tar/)) {
       let includeVcsIgnore = context.flags['include-vcs-ignore']
-      let command = tar + ' cz -C ' + dir + ' --exclude .git --exclude .gitmodules .'
+      let command = tar + ' cz -C ' + dir
 
       if (!includeVcsIgnore) {
         command += ' --exclude-vcs-ignores'
       }
+      command += ' --exclude .git --exclude .gitmodules .'
```

Comment:
Can you split this in two so the second line reads `command += ' .'`? It's quite easy to miss the positional argument right now.

Reply:
Done.